IOMC/EventVertexGenerators/python/VtxSmearedParameters_cfi.py

```diff
@@ -636,6 +650,30 @@
     Y0 = cms.double(0.1654),
     Z0 = cms.double(3.2528)
 )
+# updated numbers for 2015 PbPb data with Z centroid from fixed beamspot fits
+RealisticHICollisionFixZ2015VtxSmearingParameters = cms.PSet(
+    Phi = cms.double(0.0),
+    BetaStar = cms.double(60.0),
+    Emittance = cms.double(1.70e-07),
+    Alpha = cms.double(0.0),
+    SigmaZ = cms.double(5.2278),
+    TimeOffset = cms.double(0.0),
+    X0 = cms.double(0.1025),
+    Y0 = cms.double(0.1654),
+    Z0 = cms.double(0.771)
```

Comment:
hi @gsfs @civanch - this is quite a large "fix" in z0.. is there some documentation behind it? Otherwise, I would suggest that we just change the existing "realistic" beam spot rather than creating a new one. Thanks.